### ISSUE TITLE
Failed: FtpsStageSpec FtpRemoveSink should remove a file

### ISSUE THREAD

**Issue by 2m** [akka/alpakka#991]
```
[info] FtpRemoveSink
[info] - should remove a file *** FAILED *** (612 milliseconds)
[info]   IOResult(1,Failure(javax.net.ssl.SSLException: bad record MAC)) was not equal to IOResult(1,Success(Done)) (CommonFtpStageSpec.scala:298)
[info]   org.scalatest.exceptions.TestFailedException:
[info]   at org.scalatest.MatchersHelper$.indicateFailure(MatchersHelper.scala:346)
[info]   at org.scalatest.Matchers$AnyShouldWrapper.shouldBe(Matchers.scala:6864)
[info]   at akka.stream.alpakka.ftp.CommonFtpStageSpec$$anonfun$6$$anonfun$apply$mcV$sp$19.apply(CommonFtpStageSpec.scala:298)
[info]   at akka.stream.alpakka.ftp.CommonFtpStageSpec$$anonfun$6$$anonfun$apply$mcV$sp$19.apply(CommonFtpStageSpec.scala:290)
[info]   at org.scalatest.OutcomeOf$class.outcomeOf(OutcomeOf.scala:85)
[info]   at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
[info]   at org.scalatest.Transformer.apply(Transformer.scala:22)
[info]   at org.scalatest.Transformer.apply(Transformer.scala:20)
[info]   at org.scalatest.WordSpecLike$$anon$1.apply(WordSpecLike.scala:1078)
[info]   at org.scalatest.TestSuite$class.withFixture(TestSuite.scala:196)
[info]   at akka.stream.alpakka.ftp.FtpsStageSpec.withFixture(CommonFtpStageSpec.scala:23)
```

https://travis-ci.org/2m/alpakka/jobs/386054111

**Comment by 2m** [comment]
There seems to be some spurious test failures in various specs caused by `javax.net.ssl.SSLException: bad record MAC` exception.

A recent one: https://travis-ci.org/akka/alpakka/jobs/393496260

**Comment by 2m** [comment]
https://travis-ci.org/akka/alpakka/jobs/396448017

**Comment by 2m** [comment]
https://travis-ci.org/akka/alpakka/jobs/403107035

**Comment by 2m** [comment]
https://travis-ci.org/akka/alpakka/jobs/438480372

```
[info] FtpMoveSink
[info] - should move a file *** FAILED *** (605 milliseconds)
[info]   IOResult(1,Failure(javax.net.ssl.SSLException: bad record MAC)) was not equal to IOResult(1,Success(Done)) (CommonFtpStageSpec.scala:310)
[info]   org.scalatest.exceptions.TestFailedException:
[info]   at org.scalatest.MatchersHelper$.indicateFailure(MatchersHelper.scala:346)
[info]   at org.scalatest.Matchers$AnyShouldWrapper.shouldBe(Matchers.scala:6864)
[info]   at akka.stream.alpakka.ftp.CommonFtpStageSpec$$anonfun$7$$anonfun$apply$mcV$sp$20.apply(CommonFtpStageSpec.scala:310)
[info]   at akka.stream.alpakka.ftp.CommonFtpStageSpec$$anonfun$7$$anonfun$apply$mcV$sp$20.apply(CommonFtpStageSpec.scala:301)
```

**Comment by 2m** [comment]
https://travis-ci.org/akka/alpakka/jobs/480834644

**Comment by 2m** [comment]
https://travis-ci.org/akka/alpakka/jobs/487365062

**Comment by gabfssilva** [comment]
https://travis-ci.org/akka/alpakka/jobs/491759833

**Comment by 2m** [comment]
https://travis-ci.org/akka/alpakka/jobs/491601987

**Comment by tgambet** [comment]
I believe this can be safely closed since #1668.

**Comment by ennru** [comment]
Thank you.
Expected to be solved with #1668.
